staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -25,7 +25,6 @@ import (
 	"time"
 
 	systemd "github.com/coreos/go-systemd/daemon"
-	"github.com/emicklei/go-restful-swagger12"
```

Comment:
This causes the failed compilation. I think your editor's go package importer didn't know how to handle this line, because the directory name is different from the package name.